### PR TITLE
dts: arm: nrf528xx_pca100xx: Increase mcuboot partition size

### PR DESCRIPTION
**Pull request by nvlsianpu**
Boot serial recovery feature consume additional
memory size so the mcuboot partition must been
expanded on cost of images partitions.

Signed-off-by: Andrzej Puzdrowski <andrzej.puzdrowski@nordicsemi.no>

### CODE REVIEW

**Review by carlescufi — APPROVED** [review]

File: dts/arm/nrf52840_pca10056.dts [hunk 1]
```diff
@@ -42,19 +42,19 @@
 
 		boot_partition: partition@0 {
 			label = "mcuboot";
-			reg = <0x000000000 0x00008000>;
+			reg = <0x000000000 0x0000C000>;
 		};
 		slot0_partition: partition@8000 {
```
Comment (galak) [inline dts/arm/nrf52840_pca10056.dts]: update node names to match partition@8000 -> partition@c000

File: dts/arm/nrf52840_pca10056.dts [hunk 2]
```diff
@@ -42,19 +42,19 @@
...
 			label = "image-0";
-			reg = <0x00008000 0x00006c000>;
+			reg = <0x0000C000 0x000069000>;
 		};
 		slot1_partition: partition@74000 {
```
Comment (galak) [inline dts/arm/nrf52840_pca10056.dts]: update names to match

File: dts/arm/nrf52840_pca10056.dts [hunk 3]
```diff
@@ -42,19 +42,19 @@
...
 			label = "image-1";
-			reg = <0x00074000 0x00006c000>;
+			reg = <0x00075000 0x000069000>;
 		};
 		scratch_partition: partition@e0000 {
```
Comment (galak) [inline dts/arm/nrf52840_pca10056.dts]: update names to match

File: dts/arm/nrf52_pca10040.dts [hunk 4]
```diff
@@ -42,15 +42,15 @@
 
 		boot_partition: partition@0 {
 			label = "mcuboot";
-			reg = <0x00000000 0x8000>;
+			reg = <0x00000000 0xc000>;
 		};
 		slot0_partition: partition@8000 {
```
Comment (galak) [inline dts/arm/nrf52_pca10040.dts]: update node names to match

File: dts/arm/nrf52_pca10040.dts [hunk 5]
```diff
@@ -42,15 +42,15 @@
...
 			label = "image-0";
-			reg = <0x00008000 0x34000>;
+			reg = <0x0000C000 0x32000>;
 		};
 		slot1_partition: partition@3c000 {
```
Comment (galak) [inline dts/arm/nrf52_pca10040.dts]: update node names to match

**Review by galak — CHANGES_REQUESTED** [review]
update node names to match

File: dts/arm/nrf52840_pca10056.dts [hunk 6]
```diff
@@ -42,19 +42,19 @@
...
 			label = "image-scratch";
-			reg = <0x000e0000 0x0001d000>;
+			reg = <0x000de000 0x0001e000>;
 		};
 
 		/*
```
Comment (mbolivar) [inline dts/arm/nrf52840_pca10056.dts]: The comment and partition below here need to be fixed too (unused portion starts at 0xfc000, not 0xfd000)

**Review by mbolivar — COMMENTED** [review]
One additional comment to what @galak said

**Review by galak — APPROVED** [review]

**Pull request merged**